docs-source/content/userguide/managing-domains/fmw-infra/_index.md

```diff
@@ -7,16 +7,16 @@ pre = "<b> </b>"
 
 Starting with release 2.2, the operator supports FMW Infrastructure domains.
 This means domains that are created with the FMW Infrastructure installer rather than the WebLogic
-installer.  These domains contain the Java Required Files (JRF) feature and are
+Server installer.  These domains contain the Java Required Files (JRF) feature and are
```

Comment:
and are Server installer -> (not exactly sure what you mean here, but here are a few suggestions:)
-> and our server installer. (Server does not need to be capitalized when standalone. If you are referring to the WLS installer, you can correctly state, "and our WebLogic Server installer.")
-> and are server installed.

Reply:
I just realized that I misread this doc correction. Your edit/fix is fine.